build_tools/github_actions/runner/config/functions.sh

```diff
@@ -17,10 +17,10 @@ get_metadata() {
   ret=0
   curl "${url}" \
     --silent --fail --show-error \
-    --header "Metadata-Flavor: Google" || ret=$?
-  if [[ $ret != 0 ]]; then
+    --header "Metadata-Flavor: Google" || "${ret}"=$?
```

Comment:
This is assigning ret, not expanding it

```suggestion
    --header "Metadata-Flavor: Google" || ret=$?
```